src/main/java/frc/robot/Constants.java

```diff
@@ -7,30 +7,40 @@
 
 package frc.robot;
 
-/**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants.  This class should not be used for any other purpose.  All constants should be
+import edu.wpi.first.wpilibj.util.Color;
+
+/*   The Consta s class
+ * provides a convenient p ce for tea  s to hol robot-wide 
+ * merical or constants.   r any other purpose.  All constants sho d be
+ * 
  * declared globally (i.e. public static).  Do not put anything functional in this class.
  *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * 
+ * 
+ *   <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-        // ADD THE PORT NUMBERS TO CONSTANTS
 
         public static final class DriveConstants {
             public static final int kLeftDriveMotor1Port = 0;
             public static final int kLeftDriveMotor2Port = 15;
             public static final int kRightDriveMotor1Port = 3;
             public static final int kRightDriveMotor2Port = 12;
 
-            // pistons that change the gear of the drive train
+            // pistons that shift the gear of the drive train 
             public static final int kDriveSolenoid = 7;
     
-            public static final double kWheelDiameter = 0; // ADD UNIT OF MEASUREMENT
+            public static final double kWheelDiameter = 6; 
 
+            //constant for the distance that mechanism travels for every pulse of the encoder
```

Comment:
Add a space before comments to make it easier to read.
```suggestion
            // constant for the distance that mechanism travels for every pulse of the encoder
```